src/test/java/com/google/sps/CommentServiceTest.java

```diff
@@ -115,6 +115,28 @@ public void disabledCommentsReturnsEmptyList() throws GeneralSecurityException,
     Assert.assertEquals(Arrays.asList(), actual);
   }
 
+  /**
+   * If a video's comments can't be accessed for any other reason (ex, processing failure), a
+   * GoogleJsonResponseException should be thrown.  
+   */
+  @Test(expected = GoogleJsonResponseException.class)
+  public void otherProcessingErrorThrowsException() throws GeneralSecurityException, IOException {
+    GoogleJsonError error =
+        createExceptionDetails(new ArrayList<>(Arrays.asList("testError")),
+            "Other error detail (not commentsDisabled or videoNotFound)");
+    
+    when(requestMock.setKey(anyString())
+        .setMaxResults(anyLong())
+        .setOrder("relevance")
+        .setTextFormat("plainText")
+        .setVideoId(VIDEO_ID)
+        .execute())
+        .thenThrow(exceptionMock);
+    when(exceptionMock.getDetails()).thenReturn(error);
```

Comment:
Is it possible to set details to be the error instead? This one of those cases where you mocking more than what you need to.

Reply:
ack but leaving as is because creating a new GoogleJsonResponseException requires making a lot of other new objects